CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# unreleased
+
+* Only parse format strings when being used with `locals()` (jingw, #225).
+* Don't override paths in pyproject.toml with empty CLI paths (bcbnz, #228).
```

Comment:
After the rebase, please add a changelog entry, including your name and PR number if you like.

Reply:
If the rebase doesn't work, please start a new branch from master, copy your changes there and open a new pull request.

Reply:
I think I fixed it now. The solution was to merge master into your branch. I'll have a look at the code now :-)